### ISSUE TITLE
Why isn't this an elm package?

### ISSUE THREAD

**Issue by lsunsi** [gdotdesign/elm-storage#1]
I was so glad to find this implementation and honestly this API looks really good.
When I saw I had to use GitHub to install it, I was puzzled.

It's not that decentralised install is a bad thing, I just got curious.
Is there anything keeping this from getting added to the elm repositories?

**Comment by gdotdesign** [comment]
It cannot be / will not be on the packages.elm-lang.org because it contains native code (JavaScript) also it contains a third party library for the cookie stuff.

There will never be a local / session storage please see this by Evan: https://github.com/elm/projects/blob/master/roadmap.md#where-is-the-localstorage-package

Also after 0.19 it will be even harder to install these kinds of packages so when it comes out I will archive all of my Elm packages including this one.

**Comment by lsunsi** [comment]
Such a quick response, thanks friend!
Sad to hear that it can't be done, but I get where Evan is coming from.
Well, I might Ports my way through it then.

One last question, why will 0.19 make it even harder? ELI5 haha

**Comment by gdotdesign** [comment]
> One last question, why will 0.19 make it even harder? ELI5 haha

Because the compiler will only compile packages with native code which are in the `https://github.com/elm` or `https://github.com/elm-explorations` github organizations.

**Comment by lsunsi** [comment]
@gdotdesign got it. Well, thanks for the quick answers and information.
I think we can close this now. Have a great day (or night, I'm not sure where you are)! :D

**Comment by gdotdesign** [comment]
:+1:
